k8s/environments/stg/common-overlay/juror-digital/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
don't you need to add this kustomiation to the stg common overlay as well?

Reply:
modified it